mcodingbot/plugins/python.py

```diff
@@ -10,6 +10,25 @@
 PEP_REGEX = re.compile(r"pep[\s-]*(?P<pep>\d{1,4}\b)", re.IGNORECASE)
 
 
+DISMISS_BUTTON_ID = "dismiss"
+
+
+def encode_dismiss_button_id(id: hikari.Snowflake) -> str:
+    return f"{DISMISS_BUTTON_ID}:{id}"
+
+
+def decode_dismiss_button_id(dismiss_button_id: str) -> hikari.Snowflake:
+    return hikari.Snowflake(dismiss_button_id.split(":")[1])
+
+
+def get_dismiss_button(id: hikari.Snowflake) -> hikari.api.ActionRowBuilder:
+    action_row = plugin.app.rest.build_action_row()
+    action_row.add_button(
+        hikari.ButtonStyle.DANGER, encode_dismiss_button_id(id)
```

Comment:
I feel like a neutral button would be better.

<img width="384" alt="Screen Shot 2022-09-12 at 7 52 09 PM" src="https://user-images.githubusercontent.com/55721663/189778281-e9fc931f-8214-4845-85ab-7120031c96da.png">